packages/nuka/src/hooks/use-forward-ref.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useRef, ForwardedRef } from 'react';
+
+export const useForwardRef = <T>(ref: ForwardedRef<T>) => {
```

Comment:
`const carouselRef = ref || defaultCarouselRef;`
At first, I tried to pass `ref` that was created from `React.forwardRef` to line 123 in `carousel.tsx`, however, there was a typescript error. (Property 'current' does not exist on type '(instance: HTMLDivElement | null) => void'.ts)
That's why I created this hook to handle null and I referenced it from [here](https://github.com/facebook/react/issues/24722).

Reply:
Makes sense. Nice find 👍